src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -170,7 +170,7 @@ const useGroupInfo = () => {
     } catch (e) {
       console.log(e);
     } finally {
-      await setSelectedUser(defaultUserData);
+      setSelectedUser(defaultUserData);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `setSelectedUser` function is not an asynchronous operation, so using `await` with it is unnecessary and could potentially lead to confusion. The change made here is correct.
```diff
- await setSelectedUser(defaultUserData);
+ setSelectedUser(defaultUserData);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->